gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -148,6 +149,23 @@ public ChannelAndRequestCounter(ManagedChannel channel, ActiveRequestCounter cou
     public int activeRequests() {
       return counter.ongoingRequestCount.get();
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Can we just use https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object- ?

Reply:
Done